src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/BaseCitizenTest.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.cmc.claimstore.tests;
+
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource("/citizen-environment.properties")
+public class BaseCitizenTest extends BaseTest {
```

Comment:
why do we need it?

Reply:
This is just from my explorations on how to complete the task. I thought we might need separate configurations for citizens and solicitors and this extra layer of abstraction provides a clean way to do it.